src/test/java/org/fundacionjala/pivotal/steps/api/APIProjectSteps.java

```diff
@@ -33,7 +35,7 @@ public void iPostANewProjectToEndpoint(final String endpoint, final Map<String,
     @And("^save the response as \"([^\"]*)\"$")
     public void saveTheResponseAs(final String variableName) {
         APICommons.saveResponse(variableName, APIRequestManager.getResponse());
-        APICommons.buildEndPoint(APIRequestManager.getResponse());
+        //APICommons.buildEndPoint(APIRequestManager.getResponse());
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)